### PR TITLE
refactor:fix nits based on go report

### PR DESCRIPTION
**Pull request by MikeLing**
Signed-off-by: Tiramisu 1993 <sabergeass@gmail.com>

**1.Describe what this PR did**
Fix `typo` and `unrecognized printf verb` based on go report of pouch in https://goreportcard.com/report/github.com/alibaba/pouch
**2.Does this pull request fix one issue?** 

None
**3.Describe how you did it**

Just address them based on go report
**4.Describe how to verify it**

run `make unit-test`
**5.Special notes for reviews**

Haven't fix the `cyclomatic complexities` on the report. Not sure if it's necessery to be addressed.

### CODE REVIEW

File: cli/pull.go [hunk 1]
```diff
@@ -170,20 +170,20 @@ func formatProgressInfo(info ctrd.ProgressInfo, isTerminal bool) string {
 		if info.Total > 0.0 {
 			bar = progress.Bar(float64(info.Offset) / float64(info.Total))
 		}
-		return fmt.Sprintf("%s:\t%s\t%40r\t%8.8s/%s\t\n",
+		return fmt.Sprintf("%s:\t%s\t%40e\t%8.8s/%s\t\n",
```
Comment (MikeLing) [inline cli/pull.go]: I'm not sure if I fix the `unrecognized printf verb 'r' ` issue in the right way. Is that about to print a float number with scientific notation in here? Maybe `%f` is better?

Comment (allencloud) [reply cli/pull.go]: I think this part is introduced by @skoo87 
Could you help to check this?

**Comment by MikeLing** [comment]
Hi @allencloud , I'm not sure why the CI broken with `/go/src/github.com/alibaba/pouch/test/api_container_stop_test.go: no such file or directory`. I haven't change anything related with it and keep even with the lastest code base I think.

**Comment by allencloud** [comment]
@Letty5411  is digging this. The failure may be related to merging a previous commit which has not been rebased to the latest commit.

**Comment by allencloud** [comment]
LGTM

**Comment by 0x04C2** [comment]
Hi @allencloud , @MikeLing , Sorry for late reply

`%r` is custom formatter. Ref link is [here](https://golang.org/pkg/fmt/#Formatter).

And I think we should not use `%f` here, because https://github.com/containerd/containerd/blob/master/progress/bar.go#L24.

**Pull request merged**